lib/api/apiUtils/bucket/bucketEncryption.js

```diff
@@ -1,6 +1,17 @@
 const { errors } = require('arsenal');
+const metadata = require('../../../metadata/wrapper');
+const kms = require('../../../kms/wrapper');
 const { parseString } = require('xml2js');
 
+/**
+ *
+ * @typedef {Object} serverSideEncryptionInfo
```

Comment:
Two tiny comments here, you can add a summary of the type above the @typedef here and the type def is basically a class so it usually has a capital first word.